core/markdown/src/main/scala/net/liftweb/markdown/TimeTest.scala

```diff
@@ -35,16 +35,16 @@ trait TimedTransformer {
     def deco():Decorator = Decorator
 
     private object lineTokenizer extends LineTokenizer {
-        override def allowXmlBlocks() = TimedTransformer.this.deco().allowVerbatimXml()
+        override def allowXmlBlocks: Boolean = TimedTransformer.this.deco().allowVerbatimXml()
```

Comment:
We dropped the parens `()` with this change where other places we leave parens in place on `def`'s that are empty. Should we leave the parens here as well?

Reply:
The case for this one is a bit different, I forgot to note it: the message was something like "overriding a method without parens in parent with one with parent is not supported anymore" : 
- `allowXmlBlocks` is defined without parens in `LineTokenizer`
- it must be so too in inherited methods. 

We can add parens in `LineTokenizer` but it felt like an other kind of PR ("normalizing code base" vs "removing warnings")